ui/pages/settings/index.js

```diff
@@ -757,9 +1057,9 @@ export default function Settings() {
             render: <Admins />,
           },
           {
-            name: 'providers',
-            title: 'Identity Providers',
-            render: <Providers />,
+            name: 'authentication',
```

Comment:
I re-named the 'providers' tab here. Let me know if this naming seems ok.

Reply:
Do you see this as the place to set things like password requirements?